### DISCUSSION TITLE
GitHub Education - impossible verification

### DISCUSSION THREAD

**Discussion by heywji** [community/community#48121] [GitHub Education]
Hi!
After some hours trying to reverify my account, I give up. Nothing works and I keep receiving the following message:

```code
You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
```
I took pictures of my Academic Affairs Office website, edited my GitHub profile(name, bio and etc), and even sent screenshots of the official profile page under the university domain. I reverified my account in the past without any problem but this time I am not sure what else to do...I cannot find an email address to get help so trying here as suggested in the email I received.

**Comment by samus-aran** [comment]
Hi @heywji 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you got on 👍
